systemtest/src/main/java/io/strimzi/systemtest/utils/kubeUtils/objects/PodUtils.java

```diff
@@ -234,28 +232,37 @@ public static void waitUntilPodLabelsDeletion(String podName, String... labelKey
     }
 
     /**
-     * Method waitUntilPodsByNameStability ensuring for every pod listed for kafka or zookeeper statefulSet will be controlling
-     * their status in Running phase. If the pod will be running for selected time #Constants.GLOBAL_RECONCILIATION_COUNT
-     * pod is considered as a stable. Otherwise this procedure will be repeat.
-     * @param podNamePrefix all pods that matched the prefix will be verified
+     * Method waitForPendingPod ensures that at least one pod from listed (by prefix) will be in Pending phase
+     * @param podPrefix - all pods that matched the prefix will be verified
      */
-    public static void waitUntilPodsByNameStability(String podNamePrefix) {
-        verifyThatPodsAreStable(() -> ResourceManager.kubeClient().listPodsByPrefixInName(podNamePrefix));
+    public static void waitForPendingPod(String podPrefix) {
+        LOGGER.info("Wait for at least one pod with prefix: {} will be in pending phase", podPrefix);
+        TestUtils.waitFor("One pod to be in PENDING state", Constants.GLOBAL_POLL_INTERVAL, Constants.GLOBAL_TIMEOUT,
+            () -> {
+                List<Pod> actualPods = kubeClient().listPodsByPrefixInName(podPrefix);
+                return actualPods.stream().anyMatch(pod -> pod.getStatus().getPhase().equals("Pending"));
+            });
     }
 
     /**
-     * * Waits until all matching pods are {@linkplain #verifyThatPodsAreStable(Supplier)} stable} in the "Running" phase.
-     * @param pods all pods that will be verified
-     */
-    public static void waitUntilPodsStability(List<Pod> pods) {
-        verifyThatPodsAreStable(() -> pods);
+     * Method waitForPodsStability ensuring for every pod listed for kafka or zookeeper statefulSet will be controlling
+     * their status in Running phase. If the pod will be running for selected time #Constants.GLOBAL_RECONCILIATION_COUNT
+     * pod is considered as a stable. Otherwise this procedure will be repeat.
```

Comment:
```suggestion
     * Ensures every pod in a StatefulSet in the the {@code Running} phase. 
```

```suggestion
     * Ensures every pod in a StatefulSet is stable in the {@code Running} phase. 
     * A pod must be in the Running phase for {@link #Constants.GLOBAL_RECONCILIATION_COUNT} seconds for
     * it to be considered as stable. Otherwise this procedure will be repeat.
```